tests/utils/test_requirements_utils.py

```diff
@@ -410,3 +411,24 @@ def predict(self, context, model_input, params=None):
     captured_modules = _capture_imported_modules(model_info.model_uri, "pyfunc")
     assert "pandas" in captured_modules
     assert "sklearn" in captured_modules
+
+
+def test_capture_imported_modules_includes_extras():
+    class MyModel(mlflow.pyfunc.PythonModel):
+        def predict(self, _, inputs):
```

Comment:
```suggestion
        def predict(self, _, inputs, params=None):
```
Let's use the method signature defined in PythonModel.